packages/core/src/agent/getOutboundMessageContext.ts

```diff
@@ -297,8 +297,11 @@ async function addExchangeDataToMessage(
     associatedRecord: BaseRecordAny
   }
 ) {
+  const legacyInvitationMetadata = outOfBandRecord?.metadata.get(OutOfBandRecordMetadataKeys.LegacyInvitation)
```

Comment:
this file has an unused import from `http` which is a mistake, but not from your PR. Can you remove it? 

GitHub does not allow me to add a comment there...